index.html

```diff
@@ -44,8 +44,8 @@ <h1 class="title">I’m Jery Mey. <br>
       </section>
       <!-- ---------- cards section ----------->
 
-      <section class="card-container" id="card-container">
-        <article class="cards cards-1">
+      <section class="card-container" id="work">
+        <!-- <article class="cards cards-1">
```

Comment:
- According to the project requirements, the content of your card sections should be rendered dynamically using JavaScript, Please fix this 🙏. All HTML code should be removed and rendered using JavaScript